packages/carousel/src/Carousel.js

```diff
@@ -1,39 +1,56 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import {
-  CarouselProvider,
-  Slider,
-  Slide,
-  ButtonBack,
-  ButtonNext,
-  DotGroup,
-} from 'pure-react-carousel'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
 import moize from 'moize'
 import { v4 as uuidv4 } from 'uuid'
 
+import { layoutToFlexWidths } from './layoutToFlexWidths'
 import { CarouselWrapper } from './Carousel.styles'
 
 const getSlideKey = moize(uuidv4)
 
 export const Carousel = ({
   children,
+  showDots = false,
```

Comment:
NB: should we define the defaults with `defaultProps` property?

Reply:
perhaps - defaultProps will render in storybook docs i think